packages/cli/src/commands/certs/auto/disable.ts

```diff
@@ -0,0 +1,36 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import confirmApp from '../../../lib/apps/confirm-app'
+import heredoc from 'tsheredoc'
+
+export default class Disable extends Command {
+  static topic = 'certs';
+  static description = 'disable ACM for an app';
+  static flags = {
+    confirm: flags.string({char: 'c'}),
```

Comment:
This flag was hidden in the v5 version. Is there a reason to not hide it now?

Reply:
Good question. Yeah I noticed it was hidden and thought maybe it was an error originally because I can't think of a product reason why we would present this as a hidden flag. I can change it back, but we might want to think about documenting some conventions around how we're presenting commands to users and then maybe comment or something when we're diverging from those conventions.

Reply:
Yeah, I'd leave it as hidden for now and we can address that as a larger question later. Good idea to call it out, though.